scim-core/src/test/java/org/apache/directory/scim/core/repository/PatchHandlerTest.java

```diff
@@ -49,6 +52,7 @@ public class PatchHandlerTest {
   public PatchHandlerTest() {
     SchemaRegistry schemaRegistry = new SchemaRegistry();
     schemaRegistry.addSchema(ScimUser.class, List.of(EnterpriseExtension.class));
+    schemaRegistry.addSchema(ScimGroup.class, null);
```

Comment:
The tests don't auto discover all the schema, so they must manually be added (this was causing one of the new tests to fail)